### ISSUE TITLE
[Bug]: bazel run to update snapshots will fail if test relies on `'TEST_SRCDIR'`

### ISSUE THREAD

**Issue by Aghassi** [aspect-build/rules_jest#96]
### What happened?

I have tests written by folks who used rules_nodejs prior and am migrating them to rules_jest. The tests specifically rely on some bazel test environment directories. When running `bazel run :test_update_snapshots`, the tests fail with `rules_jest` because these bazel environment variables are not exported. 

For example, `TEST_SRCDIR` doesn't exist, but I could tap `JS_BINARY__RUNFILES`, which seems to be the equivalent.

List of full envs that could be expected found here https://bazel.build/reference/test-encyclopedia

### Version

Development (host) and target OS/architectures:

Output of `bazel --version`: 6.0.0

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file: rules_jest - 0.14.4

Language(s) and/or frameworks involved:

### How to reproduce

```shell
Have a test that does something like 



const SRCDIR = join(process.env['TEST_SRCDIR']!, 'folder')


in it and run `bazel run :test_update_snapshots`. It will fail.
```

### Any other information?

This also impacts anything that needs `TEST_TMPDIR`

### Fund our work

- [ ] Sponsor our open source work by donating a [bug bounty](https://opencollective.com/aspect-build/)

**Comment by gregmagolan** [comment]
Right. Since the update snapshot target is a build action (a js_run_binary under the hood), there is no `TEST_SRCDIR` set in that context. I think we should fix this by setting it explicitly there.
